src/Provider/Provider.Web/Configuration/AuthenticationConfiguration.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Esfa.Recruit.Provider.Web.Configuration
+{
+    public class AuthenticationConfiguration
+    {
+        public const int SessionTimeoutMinutes = 30;
+
+        public bool IsEnabledForDev { get; set; } = false;
```

Comment:
Would this bool property not be false by default?

Reply:
I've actually removed this switch from provider. I didn't think that anyone actually used it.